servicenow/servicenow.js

```diff
@@ -0,0 +1,164 @@
+                        //get status of ticket
+                                {                                
+                                var numbers=x.queryResult.parameters.number + "";
+                                var ci = new GlideRecord("incident");
+                                ci.addQuery("number",'ENDSWITH', numbers);
+                                ci.query();
+                                
+                                        if (ci.next()) {
+
+
+                                        if(ci.assigned_to!='')
+
+
+                                                assigned_to=ci.getDisplayValue('assigned_to');
+
+
+                                        else
+
+
+                                                assigned_to="no one";
+
+
+                                        message="Incident "+ci.number+" is currently assigned to "+assigned_to+". Current state of the incident is ''"+ci.getDisplayValue('state')+"''. This incident was last updated by "+ci.sys_updated_by+" on "+ci.sys_updated_on+". If you would like, you can ask for the update from "+assigned_to+" by updating additional comments.";
+
+
+                                        //summary={}
+
+
+                                                context='success';
+
+
+                                        }
+
+
+                                }
+                                hdrs['Content-Type'] = 'application/json';
+
+
+                                response.setStatus(200);
+
+
+                                response.setHeaders(hdrs);
+
+
+                                var response_body = {
+
+
+                                        "fulfillmentText": message,
+
+
+                                        "payload":{
+
+
+                                                "google": {
+
+
+                                                        "expectUserResponse": true,
+
+
+                                                        "richResponse": {
+
+
+                                                                "items": [
+
+
+                                                                {
+
+
+                                                                        "simpleResponse": {
+
+
+                                                                                "textToSpeech": message
+
+
+                                                                        }
+
+
+                                                                }
+
+
+                                                                ]
+
+
+                                                        }
+
+
+                                                }
+
+
+                                        }                                        
+
+
+                                };
+
+
+                                writer.writeString(global.JSON.stringify(response_body));
+
+
+                                //return response_body;
```

Comment:
Is this still necessary? I think the code is self-explained.

Reply:
this is needed for the dialogflow response

Reply:
I mean the comment "// return response_body".

Reply:
removed